apps/src/templates/rubrics/LearningGoal.jsx

```diff
@@ -38,6 +57,78 @@ export default function LearningGoal({
     setIsOpen(!isOpen);
   };
 
+  const handleFeedbackChange = event => {
+    if (studentLevelInfo.user_id && learningGoal.id) {
+      if (autosaveTimer.current) {
+        clearTimeout(autosaveTimer.current);
+      }
+      teacherFeedback.current = event.target.value;
+      setDisplayFeedback(teacherFeedback.current);
+      autosaveTimer.current = setTimeout(() => {
+        autosave();
+      }, saveAfter);
+    }
+  };
+
+  const autosave = () => {
+    setAutosaved(false);
+    setIsAutosaving(true);
+    setErrorAutosaving(false);
+    const bodyData = JSON.stringify({
+      studentId: studentLevelInfo.user_id,
+      learningGoalId: learningGoal.id,
+      feedback: teacherFeedback.current,
+      understanding: understandingLevel.current,
```

Comment:
Looking at [the tech spec](https://docs.google.com/document/d/1saBTPhEywbH1pF5ePPW9qehFDa7dGQpm3RfcUZ0QpV8/edit), I believe the `understanding` column should actually hold an understanding level as opposed to the id of an evidence level. Now that I'm thinking about it, I do wonder if the id of an evidence level makes more sense though. @wilkie -- curious if you have any thoughts here. The understanding level might make more sense on the AI side, maybe, and we'd like to keep them consistent.

Either way, I'm not seeing the correct radio button selected when I refresh the page after assigning an evidence level and I'm not sure why 🤔 The written feedback does load as expected